README.md

```diff
@@ -10,7 +10,9 @@ to the right place. In the future, we will ship `calyx` as a Rust crate and
 this won't be a problem.
 
 ### Setup Build Environment
+
 You need [Rust](https://www.rust-lang.org/install.html) and [Node.js and NPM](https://www.npmjs.com/get-npm) installed.
+To get [WebAssembly support](https://rustwasm.github.io/wasm-pack/book/quickstart.html), the easiest way is to install Rust with [rustup](https://rustup.rs) as opposed to a package manager.
```

Comment:
FWIW, I had to switch from Homebrew Rust to rustup to make this work, even with the manual `wasm-pack build` step.